fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1676,6 +1704,7 @@ void ReadYourWritesTransaction::clear( const KeyRef& key ) {
 	}
 	
 	KeyRangeRef r = singleKeyRange( key, arena );
+	approximateSize += r.expectedSize() + sizeof(KeyRangeRef);
```

Comment:
This will need to account for the conflict range too if `addWriteConflict == true`.

Reply:
Good catch!